### PR TITLE
VACMS-2215 Update metatag to increase description truncation to 320.

### PR DESCRIPTION
**Pull request by swirtSJW**
## Description

See #2215

## QA steps

As an admin:
1. Go to `/node/5977/edit`
2. In the Meta Description field paste in which contains 320 chars.
```
I am a sentence that has 100 characters in my space and happily take up and will do no more or less. I am a sentence that has 100 characters in my space and happily take up and will do no more or less. I am a sentence that has 100 characters in my space and happily take up and will do no more or less. I have 20 chars!?
```
3. Save the node. 
4. Edit the node 
- [ ] Validate that the Meta Description field contains the full text you pasted in and the ? is at the end.

### CODE REVIEW

File: composer.lock [hunk 1]
```diff
@@ -8395,6 +8395,9 @@
                         "status": "covered",
                         "message": "Covered by Drupal's security advisory policy"
                     }
+                },
+                "patches_applied": {
+                    "Fix broken fieldsets on forms": "https://www.drupal.org/files/issues/2020-05-25/3138609-9.patch"
```
Comment (swirtSJW) [inline composer.lock:8400]: Came in from another PR that did not run `lando composer update --lock`

Comment (ethanteague) [reply composer.lock:8400]: My bad, @swirtSJW :+1:

**Comment by swirtSJW** [comment]
devshop rebuild

**Comment by swirtSJW** [comment]
devshop go

**Comment by swirtSJW** [comment]
bleep off devshop

**Comment by swirtSJW** [comment]
closing to trick devshop

**Comment by swirtSJW** [comment]
CLosing this to open a new PR that is identical.   Just to get devshop to build it.

**Pull request closed without merging**